src/cohort.js

```diff
@@ -0,0 +1,101 @@
+class Cohort {
+  constructor() {
+    this.cohortList = []
+  }
+
+  createCohort(cohortName) {
+    if (!cohortName) {
+      throw new Error('Please give a name to your cohort!')
+    }
+
+    const findCohort = this.cohortList.find(
```

Comment:
Try to name variables as nouns and functions as verbs. `findCohort` suggests that this is a function used to find a cohort, a better name would be `foundCohort`

Reply:
Ok, will do the changes.